takari-lifecycle-plugin/src/main/java/io/takari/maven/plugins/testproperties/TestPropertiesMojo.java

```diff
@@ -267,7 +268,7 @@ private static void putIfAbsent(Properties properties, String key, String value)
   private String expand(String value, Map<Object, Object> substitutes) {
     StringWriter writer = new StringWriter();
     try {
-      resourceProcessor.filter(new StringReader(value), writer, substitutes);
+      resourceProcessor.filter(new StringReader(value), writer, substitutes, MissingPropertyAction.empty);
```

Comment:
why not `fail`? I've been bitten more than once by tests mysteriously failing due to missing test properties. (to be clear, I am not asking to change the code, but want to understand the reason you implemented like you did).

Reply:
Just wanted to retain "default" behaviour, no other reason. I do like to have "fail" here too, but did not want to disrupt anyone.